### ISSUE TITLE
Updating 1.4 -> 1.5 causes issues

### ISSUE THREAD

**Issue by OmgImAlexis** [babel/babel-preset-env#335]
I don't directly use `babel-preset-env` but this shouldn't really be happening. I removed my `node_modules` today and ran `yarn install` now I'm getting this error. Forcing 1.4.0 to be used seems to have "fixed" the issue for now.

Where exactly should I report this? I'm on `node v7.10.0`.

```
/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:328
        throw e;
        ^

Error: Invalid version passed for target "node": "current.0.0". Versions must be in semver format (major.minor.patch) (While processing preset: "/Users/xo/code/Medusa/vue/node_modules/babel-preset-env/lib/index.js")
    at /Users/xo/code/Medusa/vue/node_modules/babel-preset-env/lib/index.js:69:13
    at Array.filter (native)
    at isPluginRequired (/Users/xo/code/Medusa/vue/node_modules/babel-preset-env/lib/index.js:59:54)
    at filterItem (/Users/xo/code/Medusa/vue/node_modules/babel-preset-env/lib/index.js:99:20)
    at Array.filter (native)
    at buildPreset (/Users/xo/code/Medusa/vue/node_modules/babel-preset-env/lib/index.js:149:56)
    at /Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:317:46
    at Array.map (native)
    at OptionManager.resolvePresets (/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
    at OptionManager.mergePresets (/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
    at OptionManager.mergeOptions (/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
    at OptionManager.init (/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/Users/xo/code/Medusa/vue/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at Array.module.exports (/Users/xo/code/Medusa/vue/node_modules/require-extension-hooks-babel/index.js:25:22)
    at hook (/Users/xo/code/Medusa/vue/node_modules/require-extension-hooks/hook.js:15:26)
    at /Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at extensions.(anonymous function) (/Users/xo/code/Medusa/vue/node_modules/require-precompiled/index.js:16:3)
    at /Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/Users/xo/code/Medusa/vue/node_modules/ava/lib/process-adapter.js:100:4)
    at Object.<anonymous> (/Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/xo/code/Medusa/vue/test/app.spec.js:6:1)
    at Module._compile (module.js:571:32)
    at Module.replacementCompile (/Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at extensions.(anonymous function) (/Users/xo/code/Medusa/vue/node_modules/require-precompiled/index.js:13:11)
    at /Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/Users/xo/code/Medusa/vue/node_modules/ava/lib/process-adapter.js:100:4)
    at Object.<anonymous> (/Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/xo/code/Medusa/vue/node_modules/ava/lib/test-worker.js:49:1)
    at Module._compile (module.js:571:32)
    at Module.replacementCompile (/Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at module.exports (/Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/default-require-extensions/js.js:8:9)
    at Object.<anonymous> (/Users/xo/code/Medusa/vue/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Function.Module.runMain (module.js:605:10)
    at Function.runMain (/Users/xo/.node-spawn-wrap-16602-afc6a61e310c/node:40:10)
    at Object.<anonymous> (/Users/xo/code/Medusa/vue/node_modules/nyc/bin/wrap.js:23:4)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Function.Module.runMain (module.js:605:10)
    at Object.<anonymous> (/Users/xo/.node-spawn-wrap-16602-afc6a61e310c/node:140:8)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:427:7)
    at startup (bootstrap_node.js:151:9)
    at bootstrap_node.js:542:3
```

**Comment by existentialism** [comment]
@OmgImAlexis thanks for the report, can you show the babelrc or config that passes options to the preset, curious how the targets are configured.

**Comment by OmgImAlexis** [comment]
```
{
    "presets": [
        "es2015",
        "stage-3"
    ],
    "plugins": [
        "transform-runtime",
        "syntax-async-functions"
    ],
    "ignore": "test.js",
    "env": {
        "development": {
            "sourceMaps": "inline"
        }
    }
}
```

**Comment by yavorsky** [comment]
@OmgImAlexis So you don't use babel-preset-env? There is no 'env' in your config preset's list.

**Comment by OmgImAlexis** [comment]
Nope none at all.

**Comment by yavorsky** [comment]
@OmgImAlexis Let's try to figure out what your dep is using babel-preset-env and passes incorrect options.
Can you please run smth like `npm list | grep --color -E 'babel-preset-env|$'` and note what of the npm packages are using `babel-preset-env`. Then we'll have an entry point.

**Comment by OmgImAlexis** [comment]
Ah, think I found the repo with the issue.
https://github.com/jackmellis/require-extension-hooks-babel

```
├─┬ require-extension-hooks-babel@0.1.0
│ └─┬ babel-preset-env@1.5.1
│   ├─┬ browserslist@2.1.4
│   │ └── caniuse-lite@1.0.30000670
│   └── semver@5.3.0
```

**Comment by OmgImAlexis** [comment]
Should that error not bubble up from the package it's being thrown in? This would at least allow the user to know what npm package is causing the issue.

**Comment by yavorsky** [comment]
@OmgImAlexis This bug was caused by require-extension-hooks-babel.
It passed invalid target's value: `node: ["current"]`. 

Suddenly it was fixed, https://github.com/jackmellis/require-extension-hooks-babel/commit/47313d0c67518ba664118d3b849e809438a15903. And then published 0.1.1.

Seems like you are still using 0.1.0. Please update it and let us know if it won't help.
